code/modules/holiday/holidays.dm

```diff
@@ -589,3 +589,11 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
 
 /datum/holiday/easter/getStationPrefix()
 	return pick("Fluffy","Bunny","Easter","Egg")
+
+/datum/holiday/ianbirthday
+	name = "Ian's Birthday"
+	begin_day = 15
+	begin_month = MARCH
```

Comment:
Ian's birthday is actually 10 Sep 2011 UTC+0. Here's the commit where errorage added corgis and Ian:
https://github.com/tgstation/tgstation/commit/de7e4f0de0d568cd6e1f0d7bcc3fd34700598acb#diff-9aa879074aa89b230889437f7c112619R61
You can compare tgstation.2.0.8.dmm or mob.dmi with their previous versions to see the absence of Ian on the map and corgi sprites in that file.

Reply:
March 15th was chosen as it's the date where Ian's age was first started to be tracked.